cmd/start.go

```diff
@@ -29,7 +30,7 @@ func init() {
 	// allows environment variables to use _ instead of -
 	viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_")) // sync-provider becomes SYNC_PROVIDER
 	viper.SetEnvPrefix("FLAGD")                            // port becomes FLAGD_PORT
-
+	flags.Int32P(metricsPortFlagName, "m", 8014, "Port to serve metrics on")
```

Comment:
Once this is merged, we should add this to the Prometheus wiki page on port allocations. 

https://github.com/prometheus/prometheus/wiki/Default-port-allocations